lib/main.coffee

```diff
@@ -2,6 +2,16 @@
 helpers = require 'atom-linter'
 path = require 'path'
 
+isGreaterVersion = (v1, v2) ->
```

Comment:
this should be called `isLessThanVersion` , as 

- equal to `0.10.0` is acceptable, and
- the name of the function should be focused on the first argument (`result`)